core/src/test/java/com/klaytn/caver/common/utils/UtilsTest.java

```diff
@@ -106,7 +108,7 @@ public void invalidPrivateKey() {
         @Test
         public void validWalletKey() {
             String walletKey = caver.wallet.keyring.generate().getKlaytnWalletKey();
-            assertTrue(caver.utils.isKlaytnWalletKey(walletKey));
+            assertTrue(Utils.isKlaytnWalletKey(walletKey));
```

Comment:
Is there any reason you changed from `caver.utils` to `Utils`?

Reply:
Thanks for review. I modified it.